app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] != "mv":
+        print("Please enter the right command")
+        return
+    if "/" in command_list[2]:
+        path_list = command_list[2].split("/")
+    if "\\" in command_list[2]:
+        path_list = command_list[2].split("\\")
+    path = ""
+    for i in range(len(path_list) - 1):
+        path = os.path.join(path, path_list[i])
+        os.mkdir(path)
```

Comment:
if the directory is already created it causes an error
`FileExistsError: [Errno 17] File exists`

Reply:
Done